### DISCUSSION TITLE
Trying to make a white box with label inside

### DISCUSSION THREAD

**Discussion by Multirious** [emilk/egui#1932] [Q&A]
I've created a discussion about this before but seems like it's not getting the attention, maybe because it's too vague. So now I'm changing the question. [Static/Max Size Container?](#1853 )
Basically I'm trying to create a widget that's white box and with some label inside.
![image](https://user-images.githubusercontent.com/77918086/185547940-438fab3f-0438-4807-a80a-d349f108a23c.png)
Like this. But I'm not sure how to do it.
I wanted the box to be a fixed size.

**Comment by emilk** [comment]
Have you looked at https://github.com/emilk/egui/blob/master/crates/egui_demo_lib/src/demo/toggle_switch.rs?
